examples/vertx-opentracing/genorders.sh

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+while :
+do
+	declare -a AccountIds=('joe' 'sarah' 'fred' 'jane' 'brian' 'steve');
+
+	declare -a ItemIds=('laptop' 'car' 'book' 'chair' 'table' 'pencil' 'dvd');
+
+	itemId=`echo ${ItemIds[$[ RANDOM % ${#ItemIds[@]} ]]}`
+	accountId=`echo ${AccountIds[$[ RANDOM % ${#AccountIds[@]} ]]}`
+
+	quantity=`echo $[ 1 + $[ RANDOM % 10 ]]`
+
+
+        date
```

Comment:
was just to have an indication of time when order was being generated - not critical, but it is only for demo.